ckanext/harvest/templates/source/admin.html

```diff
@@ -6,6 +6,12 @@
     {% if source.status and source.status.last_job %}
       {% snippet "snippets/job_details.html", job=source.status.last_job %}
       <div class="form-actions">
+        {% if source.status and source.status.last_job and (source.status.last_job.status == 'Running') %}
+        <a href="{{ h.url_for('harvest_job_abort', source=source.name, id=source.status.last_job.id) }}" class="btn" title="Stop this Job">
+          <i class="icon-ban-circle"></i>
+          {{ _('Stop') }}
+        </a>
+        {% endif %}
```

Comment:
Instead of putting this new button in the bottom part next of the "View full job report", I would rather put it next to the other actions in the upper part between the "Reharvest" and the "Clear" action.

Reply:
👍  My initial inclination was, all the above buttons are for actions related to the source and the below is for the last job.  I'll move it between Reharvest and Clear.

Reply:
I understand and that could make sense as well. But e.g. the reharvest depends on the last job as well, as you can't start a new one, if one is currently running.